packages/select/src/Select/components/Trigger/TriggerButtonBase/TriggerButtonBase.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { HTMLAttributes } from "react"
+import { useFocusRing } from "@react-aria/focus"
+import { mergeProps } from "@react-aria/utils"
+import classNames from "classnames"
+import { OverrideClassName } from "@kaizen/component-base"
+import { Icon } from "@kaizen/component-library"
+import chevronDown from "@kaizen/component-library/icons/chevron-down.icon.svg"
+import chevronUp from "@kaizen/component-library/icons/chevron-up.icon.svg"
+
+import { useMenuTriggerContext } from "../../../provider/MenuTriggerProvider"
+import styles from "./TriggerButtonBase.module.scss"
+
+export interface TriggerButtonBaseProps
+  extends OverrideClassName<HTMLAttributes<HTMLDivElement>> {
```

Comment:
should it be button element type ?
```suggestion
  extends OverrideClassName<HTMLAttributes<HTMLButtonElement>> {
```